src/handlers/motions/motionCreated/handlers/metadataDelta.ts

```diff
@@ -0,0 +1,53 @@
+import { BigNumber } from 'ethers';
+import { TransactionDescription } from 'ethers/lib/utils';
+import { ColonyActionType } from '~graphql';
+import { ContractEvent } from '~types';
+import {
+  getPendingMetadataDatabaseId,
+  isAddVerifiedMembersOperation,
+  parseOperation,
+  verbose,
+} from '~utils';
+import { createMotionInDB } from '../helpers';
+
+export const handleMetadataDeltaMotion = async (
+  event: ContractEvent,
+  desc: TransactionDescription,
+  gasEstimate: BigNumber,
```

Comment:
Not related to this PR, but the gas estimate shouldn't need passing around like this.

I guess we'll need to refactor this out at some point.